tests/common/rpc_api.py

```diff
@@ -0,0 +1,136 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+from base64 import b64encode
+import pytest
+from typing import Dict, Optional, Callable
+from quart.typing import TestClientProtocol
+from werkzeug.datastructures import Headers
+
+from parsec._parsec import DateTime, OrganizationID
+from parsec.api.version import API_VERSION
+from parsec.core.types import LocalDevice
+from parsec.api.protocol.ping import ping_serializer
+
+from tests.common import RunningBackend, OrganizationFullData
+
+
+class AuthenticatedRpcApiClient:
+    def __init__(self, client: TestClientProtocol, device: LocalDevice):
+        self.client = client
+        self.device = device
+        self.base_headers = Headers(
+            {
+                "Content-Type": "application/msgpack",
```

Comment:
```suggestion
                "Content-Type": CONTENT_TYPE_MSGPACK,
```

Reply:
can you revert those changes ? I think it's better to keep those value separated from the constant in the codebase

The idea is to be able to detect a backward incompatibility if we change by mistake the constants

Reply:
done